hack/install-tools.sh

```diff
@@ -52,16 +61,23 @@ if [ -z "$PY_CMD" ]; then
     fi
 fi
 
-ALT_BIN="$(realpath "${2:-.bin}")"
+ALT_BIN="${2:-.bin}"
 case "$1" in
-    --gitlint)
+    "--yamllint")
```

Comment:
May be you could add `--yamllint` to the help section listed towards the end of script.
```diff --git a/hack/install-tools.sh b/hack/install-tools.sh
index 61aec4d..2c7bc48 100755
--- a/hack/install-tools.sh
+++ b/hack/install-tools.sh
@@ -74,5 +74,6 @@ case "$1" in
         echo "available tools:"
         echo "  --gitlint"
         echo "  --shellcheck"
+        echo "  --yamllint"
     ;;
 esac
```

Reply:
good catch! done, thanks!